article/test/ArticleMetaDataTest.scala

```diff
@@ -14,20 +14,30 @@ import play.api.test.Helpers._
     val body = Jsoup.parseBodyFragment(contentAsString(result))
     status(result) should be(200)
 
-    val script = body.getElementsByAttributeValue("data-schema", "organization")
+    val script = body.getElementsByAttributeValue("data-schema", "Organization")
     script.size() should be(1)
 
-    val data: JsValue = Json.parse(script.first().html())
-    val organisation = data.asInstanceOf[JsArray](0)
-    val appIndexer = data.asInstanceOf[JsArray](1)
+    val organisation: JsValue = Json.parse(script.first().html())
```

Comment:
it specifically looks for the Organization section, if we have more than one organisation then it's a bug, so the assertion on line 18 will fail